libs/interpret-vision/src/lib/VisionExplanationDashboard/VisionExplanationDashboard.tsx

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  IDropdownOption,
+  Dropdown,
+  Text,
+  Stack,
+  Pivot,
+  PivotItem,
+  SearchBox,
+  Slider,
+  CommandBarButton
+} from "@fluentui/react";
+import { localization } from "@responsible-ai/localization";
+import React from "react";
+
+import { ImageList } from "./Controls/ImageList";
+import { TitleBar } from "./Controls/TitleBar";
+import { IVisionExplanationDashboardProps } from "./Interfaces/IExplanationDashboardProps";
+import { visionExplanationDashboardStyles } from "./VisionExplanationDashboardStyles";
+
+export interface IVisionExplanationDashboardState {
+  imageDim: number;
+  panelOpen: boolean;
+  selectedKey: string;
+}
+
+enum VisionDatasetExplorerTabOptions {
+  First = "Image explorer view",
+  Second = "Table view",
+  Third = "Data characteristics"
+}
+
+export enum TitleBarOptions {
+  Error,
+  Success
+}
+
+export class VisionExplanationDashboard extends React.Component<
+  IVisionExplanationDashboardProps,
+  IVisionExplanationDashboardState
+> {
+  public constructor(props: IVisionExplanationDashboardProps) {
+    super(props);
+
+    this.state = {
+      imageDim: 200,
+      panelOpen: false,
+      selectedKey: VisionDatasetExplorerTabOptions.First
+    };
+  }
+
+  public render(): React.ReactNode {
+    const classNames = visionExplanationDashboardStyles();
+    const DropdownOptions: Array<IDropdownOption<unknown>> = [
+      { key: 0, text: "All Data" }
```

Comment:
localize if visible to user - I believe this constant is already available in the localization file